serde/src/test/org/apache/hadoop/hive/serde2/objectinspector/TestObjectInspectorUtils.java

```diff
@@ -76,7 +76,7 @@ public void testObjectInspectorUtils() throws Throwable {
           .getStandardObjectInspector(oi1);
       List<? extends StructField> fields = soi.getAllStructFieldRefs();
       assertEquals(10, fields.size());
-      assertEquals(fields.get(0), soi.getStructFieldRef("aint"));
+      assertEquals(fields.get(0), soi.getStructFieldRef("astring"));
```

Comment:
In addition to sorting for ensuring proper order of `getDeclaredFields`, why is this change needed

Reply:
The assertion should be changed because the first element has been changed after sorting.